### PR TITLE
Literals: Add a link to command literals

### PR DESCRIPTION
**Pull request by alexherbo2**

### CODE REVIEW

File: syntax_and_semantics/literals.md [hunk 1]
```diff
@@ -20,3 +20,4 @@ Crystal provides several literals for creating values of some basic types.
 | [Tuple](./literals/tuple.html)            | `{1, "hello", 'x'}`                         |
 | [NamedTuple](./literals/named_tuple.html) | `{name: "Crystal", year: 2011}`, `{"this is a key": 1}`|
 | [Proc](./literals/proc.html)              | `->(x : Int32, y : Int32) { x + y }`        |
+| [Command](./literals/command.html) | `` `echo foo` `` |
```
Comment (straight-shoota) [inline syntax_and_semantics/literals.md]: Could you please adjust the padding that the `|` character is on the same column as in the previous lines?

Comment (alexherbo2) [reply syntax_and_semantics/literals.md]: I’m waiting #346 to be merged to adjust the proper padding.

Comment (bcardiff) [reply syntax_and_semantics/literals.md]: #346 merged

Comment (bcardiff) [reply syntax_and_semantics/literals.md]: I kind of like that literal commands can work with interpolation, as shown in #347.

**Comment by alexherbo2** [comment]
@straight-shoota Can be merged now.

**Pull request merged**